### ISSUE TITLE
Created env having all general python libraries already installed

### ISSUE THREAD

**Issue by Ugomartinez** [conda/conda#14293]
### Checklist

- [x] I added a descriptive title
- [x] I searched open reports and couldn't find a duplicate

### What happened?

I'm not very used to conda, so don't hesitate to ask if you need more precise description. 

When I used conda to create a new python environment, sometimes I end up with all the libraries of my general python installed directly in the environment. This happens when running the following commandes.
`conda create -n test python=3.10`
`conda activate test`
`wich pip` returns `/home/ugomartinez/miniconda3/envs/test/bin/pip`
`pip list` return a lot of librairies including , for example, ipywidgets, ipywidgets or pytest. 

The list returned is the same as when I'm running `pip list` while being out of any conda environment. 

This happens if I run `conda create -n test python=3.10`, but not if I run `conda create -n test_2 python=3.9` or for other version than 3.10. Then I only see pip, setuptools andwheel.

I guess the fact that my general python installed on my computer is the 3.10 plays an important role here. 
 

### Conda Info
```
     active environment : test4
    active env location : /home/ugomartinez/miniconda3/envs/test4
            shell level : 2
       user config file : /home/ugomartinez/.condarc
 populated config files : 
          conda version : 24.7.1
    conda-build version : not installed
         python version : 3.12.4.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=zen4
                          __conda=24.7.1=0
                          __cuda=12.6=0
                          __glibc=2.35=0
                          __linux=6.9.3=0
                          __unix=0=0
       base environment : /home/ugomartinez/miniconda3  (writable)
      conda av data dir : /home/ugomartinez/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/ugomartinez/miniconda3/pkgs
                          /home/ugomartinez/.conda/pkgs
       envs directories : /home/ugomartinez/miniconda3/envs
                          /home/ugomartinez/.conda/envs
               platform : linux-64
             user-agent : conda/24.7.1 requests/2.32.3 CPython/3.12.4 Linux/6.9.3-76060903-generic pop/22.04 glibc/2.35 solver/libmamba conda-libmamba-solver/24.7.0 libmambapy/1.5.8 aau/0.4.4 c/. s/. e/.
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Conda list
```
# packages in environment at /home/ugomartinez/miniconda3/envs/test:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
bzip2                     1.0.8                h5eee18b_6    defaults
ca-certificates           2024.9.24            h06a4308_0    defaults
ld_impl_linux-64          2.40                 h12ee557_0    defaults
libffi                    3.4.4                h6a678d5_1    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libgomp                   11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
libuuid                   1.41.5               h5eee18b_0    defaults
ncurses                   6.4                  h6a678d5_0    defaults
openssl                   3.0.15               h5eee18b_0    defaults
pip                       24.2            py310h06a4308_0    defaults
python                    3.10.14              h955ad1f_1    defaults
readline                  8.2                  h5eee18b_0    defaults
setuptools                75.1.0          py310h06a4308_0    defaults
sqlite                    3.45.3               h5eee18b_0    defaults
tk                        8.6.14               h39e8969_0    defaults
tzdata                    2024a                h04d1e81_0    defaults
wheel                     0.44.0          py310h06a4308_0    defaults
xz                        5.4.6                h5eee18b_1    defaults
zlib                      1.2.13               h5eee18b_1    defaults
```
### Additional Context

![Image](https://github.com/user-attachments/assets/5dd61131-9b6f-4bda-946c-8e2ca6d3ed21)

**Comment by travishathaway** [comment]
@Ugomartinez,

I have a feeling that this could be caused by Python's [Per user site-packages directory](https://peps.python.org/pep-0370/). Have you ever run `pip install --user <package>` with your system Python? Perhaps you could check the location manually to verify this? That would probably be `~/.local/lib/python3.10/site-packages/` on your computer.

`pip list` would show these packages along with the packages available in your conda environment when the Python versions match.

**Comment by Ugomartinez** [comment]
@travishathaway, 

You're right; I’ve never used ```pip install --user <package>```, but the path you mentioned is filled with various packages. It’s possible they were installed using a different flag. I’ve confirmed that ```pip list``` accurately shows what’s installed there.

I don’t have a strong interest in the specifics, but I’m concerned about whether my computer relies on any of these packages for essential functions. Is there a straightforward way to remove everything without running into major issues?

Alternatively, could you guide me on how to use Conda to avoid this situation? I’d like to create a Python 3.10 conda environment without inheriting all those packages installed by pip.

Thanks for your help!

**Comment by Ugomartinez** [comment]
Sorry, did not wanted to close it yet

**Comment by travishathaway** [comment]
@Ugomartinez,

This is actually a problem I have personally tried to solve 😄 (see: https://github.com/conda/conda/pull/13635). We wanted to make ignoring this directory a setting in conda. Unfortunately, that pull request was never accepted and merged.

My advice for you would be to place a file called `pyvenv.cfg` in the root of the new conda environment you create. You can find this location by running `conda info` when the environment is active. It's the value under `active env location`.

Inside this file, place the following line:

```ini
include-system-site-packages = false
```

This will instruct Python (and `pip` too if I'm not mistaken) to ignore the user site packages for that specific environment.

I hope this is clear. Please let me know if have any questions.

**Comment by tendermonster** [comment]
> [@Ugomartinez](https://github.com/Ugomartinez),
> 
> This is actually a problem I have personally tried to solve 😄 (see: [#13635](https://github.com/conda/conda/pull/13635)). We wanted to make ignoring this directory a setting in conda. Unfortunately, that pull request was never accepted and merged.
> 
> My advice for you would be to place a file called `pyvenv.cfg` in the root of the new conda environment you create. You can find this location by running `conda info` when the environment is active. It's the value under `active env location`.
> 
> Inside this file, place the following line:
> 
> include-system-site-packages = false
> 
> This will instruct Python (and `pip` too if I'm not mistaken) to ignore the user site packages for that specific environment.
> 
> I hope this is clear. Please let me know if have any questions.

hi there :) im facing same issue. Symtoms check out. Though I was unable to find the code that is responsable for reading using that file. If you by any chance know where that logic resides let me know :=)

**Comment by travishathaway** [comment]
@tendermonster,

The logic that is responsible for reading the `pyvenv.cfg` resides in the source code of Python itself:

- https://github.com/python/cpython/blob/8eebe4e6d02bb4ad3f1ca6c52624186903dce893/Lib/site.py
